konnectd/pkg/flagset/flagset.go

```diff
@@ -167,7 +167,7 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
 		&cli.StringFlag{
 			Name:        "iss",
 			Usage:       "OIDC issuer URL",
-			EnvVars:     []string{"KONNECTD_ISS"},
+			EnvVars:     []string{"KONNECTD_ISS", "OCIS_URL"}, // KONNECTD_ISS takes precedence over OCIS_URL
```

Comment:
While digging through the code I noticed, that this enables easy configuration for environment variable only. This will not picked up if you run `ocis --ocis-url https://ocis.owncloud.test server`.

For that to work something like for `OCIS_LOG_LEVEL` must be done. This is implemented like this:

https://github.com/owncloud/ocis/blob/d08958619e7d7022aa72de4eaa1913c52c9b0e06/ocis/pkg/command/accounts.go#L41-L60

I'm not sure which is the way to go because it also has downsides. In this example `OCIS_LOG_LEVEL` will always win, even if your provide a lower `ACCOUNTS_LOG_LEVEL`

Reply:
yes. the downside is known but accepted because the ocis binary is a poor mans orchestrator. I would argue that in eg docker compose deployments we should run the services in their own docker containers, which is why the multi repo setup was building a docker container for every service ... dunno if we still do that ...

Reply:
we'll add the `--ocis-url` flag

Reply:
scratch that ... I thought we could add a `--ocis-url` flag to the ocis server command and use `configure<Service>()` to apply it to the service config, similar to the log level and pretty flags. But that would introduce specific code that only handles this flag, which seems a little brittle to me. @IljaN mentioned that there might be something in the urfave/cli / micro/cli flags to define a flag that can be reused. So I'd postpone a `--ocis-url` cli flag until later.